### ISSUE TITLE
bad character escape sequence bug

### ISSUE THREAD

**Issue by devmondo** [fuse-box/fuse-box#54]
this is my file
`export default typeof window === `undefined`;`

and this is error i get
```
Acorn error: Bad character escape sequence (3565:26)
File: C:\projects\test\src\server\ice.js

3563  
3564  });
3565  ___scope___.file("client\utilities\is-server.js", function(exports, require, module, __filename, __dirname){ 
3566  
3567  "use strict";
3568  Object.defineProperty(exports, "__esModule", { value: true });
3569  exports.default = typeof window === "undefined";
```

**Comment by nchanged** [comment]
Can't reproduce it
`export default typeof window ===undefined;` does not emit any errors.

**Comment by nchanged** [comment]
I tried it with typescript as well. Maybe you copy paste it to make sure you got no wrong symbols there? 

http://stackoverflow.com/questions/33873982/es6-bad-character-escape-sequence-creating-ascii-string

Are you sure it's related to that line?

**Comment by devmondo** [comment]
no i did not copy paste it, i wrote the code by myself, this is really weird, why it works with previous version and not latest? i did not change my code!

**Comment by devmondo** [comment]
also does it have anything to do with the fact i am on Windows OS?

**Comment by nchanged** [comment]
Might be. But if you remove that line everything gets back to normal, right?

**Comment by devmondo** [comment]
yep!

**Comment by nchanged** [comment]
Ok, please check which version broke it. `1.3.24` does not seem to have any significant changes to the code.

**Comment by devmondo** [comment]
ok, i tried that, and everything works fine until `v1.3.33`, once i use `1.3.34` i get same error

**Comment by devmondo** [comment]
and after more tests, i found that the problem happens when this file is in an imported bundle, so if i import it directly there is zero issues, in other words, it happens when using bundle in a bundle.

**Comment by nchanged** [comment]
it might be related to this [line](https://github.com/fuse-box/fuse-box/blob/master/src/FileAnalysis.ts#L79)

Can you replace locally it with [this](https://github.com/fuse-box/fuse-box/blob/0533afa426789b7188c43bb447a6cb33c2c5ae85/src/FileAnalysis.ts#L79) version and  see if it helps

**Comment by nchanged** [comment]
Fixed in `1.3.38`

**Comment by devmondo** [comment]
fixed thanks :)
